hbase_storage/src/main/java/com/splicemachine/hbase/ZkUtils.java

```diff
@@ -315,10 +317,17 @@ public static long nextSequenceId(String counterNode) throws IOException{
 		 */
 
         RecoverableZooKeeper rzk=ZkUtils.getRecoverableZooKeeper();
-        int maxTries=10;
+        int maxTries=Integer.MAX_VALUE;
```

Comment:
Is this common in zk code? I'm not very familiar with this but loop forever gives a strange feel...

Reply:
Retry until successful.